packages/Utils/test/tstMiscellaneous.cpp

```diff
@@ -16,10 +16,10 @@
 TEUCHOS_UNIT_TEST( DataTransferKitRuntimeAPI, return_version )
 {
     auto const dtk_version = DataTransferKit::version();
-    TEUCHOS_ASSERT( !dtk_version.empty() );
+    TEST_ASSERT( !dtk_version.empty() );
```

Comment:
I had these changes sitting on some branch :)
Good catch.  Did you actually grep for `TEUCHOS_`?

Reply:
No, I got a warning that `out` and `success` were not used.

Reply:
Please do so and change if there is any other instance.

Reply:
I just grepped and there shouldn't be any other `TEUCHOS_ASSERT`